packages/config/src/load.js

```diff
@@ -27,6 +29,16 @@ export async function loadNuxtConfig ({
     configFile = undefined
   }
 
+  // Load env
+  const env = loadEnv(rootDir)
+
+  // Fill process.env so it is accessible in nuxt.config
+  for (const key in env) {
+    if (key[0] !== '_' && process.env[key] === undefined) {
```

Comment:
nit for better readability
```suggestion
    if (!key.startsWith('_') && process.env[key] === undefined) {
```